__tests__/services/fhir.spec.ts

```diff
@@ -14,21 +25,93 @@ import {
     getIncludedResource,
     getIncludedResources,
     getConcepts,
+    applyFHIRService,
+    applyFHIRServices,
 } from '../../services/fhir';
-
 import { service } from '../../services/service';
 import { success, failure } from '../../libs/remoteData';
 import { Bundle, AidboxResource } from 'src/contrib/aidbox';
+import { AxiosTransformer } from 'axios';
 
 jest.mock('../../services/service', () => {
     return { service: jest.fn(() => Promise.resolve(success('data'))) };
 });
 
-describe('Service `fhir`', () => {
+describe.only('Service `fhir`', () => {
     beforeEach(() => {
         jest.clearAllMocks();
     });
 
+    describe('method `create`', () => {
+        test('create resource without search parameters`', async () => {
+            const resource = {
+                resourceType: 'Patient',
+            };
+
+            expect(create(resource)).toEqual({
+                method: 'POST',
+                url: `/${resource.resourceType}`,
+                data: resource,
+            });
+        });
+
+        test('create resource with search parameters`', () => {
+            const resource = {
+                resourceType: 'Patient',
+            };
+            const searchParams = {
+                param: 'value',
+            };
+
+            expect(create(resource, searchParams)).toEqual({
+                method: 'POST',
+                url: `/${resource.resourceType}`,
+                data: resource,
+                headers: {
+                    'If-None-Exist': searchParams,
+                },
+            });
+        });
+    });
+
+    test('method `createFHIRResource`', async () => {
+        const reference = {
```

Comment:
Create should receive resource, not reference. Let's name it as `reference` to be more correct.